mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule Paironauts.Mixfile do
     [
       app: :paironauts,
       version: "0.0.1",
-      elixir: "1.6.3",
+      elixir: "~> 1.6",
```

Comment:
I thought you were right on when you proposed this the other day, but now I'm thinking I want us to keep the version pinned like it is.  I put some comments on @kenshinji's #19 about this. 

You are responding to a shortcoming with the project that I agree needs to be resolved.  I want to make it easy for devs to get spun up, but I'd really like to keep the Elixir version a bit stricter. Do you have any thoughts on how we might do that? Or why it might not be important?

Reply:
The official language docs use `brew install elixir` as the official install instructions, and that's personally what I use.  I haven't yet been forced to use a version manager for Elixir, and I'd prefer to hold off on that as long as possible.  Unless you want to make using a Elixir versioning tool a prerequisite, I'd recommend making the Elixir versioning more flexible.

Reply:
I do think we can trust Elixir to follow good SemVer practices well. What I would like to see happen for this project is for every dependency, including language, to be pinned. To make a blanket statement, I think this is a good practice with few drawbacks. [Kiex](https://github.com/taylor/kiex) is an apparently mature and trustworthy tool for this that no contributors have had trouble with yet. It's easy to set up.

It's fine for the scope of this PR. I will restate my personal preference that in the future we pin the version and add version manager install instructions to the docs. Anyone else have thoughts on the subject?